docs/releases/1.9.2.txt

```diff
@@ -30,3 +30,7 @@ Bugfixes
   ``db_index=True`` or ``unique=True`` to a ``CharField`` or ``TextField`` that
   already had the other specified, or when removing one of them from a field
   that had both (:ticket:`26034`).
+
+* Fixed a regression with abstract models defining application relative lazy
```

Comment:
This is a tough one to describe, but here's my try: "Fixed a regression where defining a relation on an abstract model's field using a string model name without an app_label no longer resolved that reference to the abstract model's app if using that model in another app"